inventory/services_test.go

```diff
@@ -394,6 +394,51 @@ func TestMySQLService(t *testing.T) {
 		}
 	})
 
+	t.Run("AddAddressSocketConflict", func(t *testing.T) {
+		t.Parallel()
+
+		genericNodeID := addGenericNode(t, pmmapitests.TestString(t, "")).NodeID
+		require.NotEmpty(t, genericNodeID)
+		defer pmmapitests.RemoveNodes(t, genericNodeID)
+
+		params := &services.AddMySQLServiceParams{
+			Body: services.AddMySQLServiceBody{
+				NodeID:      genericNodeID,
+				Address:     "localhost",
+				Port:        3306,
+				Socket:      "/var/run/mysqld/mysqld.sock",
+				ServiceName: pmmapitests.TestString(t, "MySQL Service with address and socket conflict"),
+			},
+			Context: pmmapitests.Context,
+		}
+		res, err := client.Default.Services.AddMySQLService(params)
+		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "setting both address and socket in once is disallowed")
```

Comment:
maybe `not allowed`

Reply:
Well... Grammarly says that current variant is OK